src/cpp/session/modules/SessionRmdNotebook.R

```diff
@@ -1138,6 +1138,20 @@ assign(".rs.notebookVersion", envir = .rs.toolsEnv(), "1.0")
    .rs.scalarListFromList(defaultOptions)
 })
 
+.rs.addFunction("executeChunkCallback", function(chunkName, chunkCode)
+{
+  if (exists(".rs.notebookChunkCallbacks") &&
+      length(.rs.notebookChunkCallbacks) != 0)
+  {
+     results <- list()
```

Comment:
Now that there's only a single callback result, it seems unnecessary to create and then append the result to an R list -- why not just return the result directly?